deltachat-ios/Controller/NewGroupController.swift

```diff
@@ -12,6 +12,7 @@ class NewGroupController: UITableViewController, MediaPickerDelegate {
     private var deleteGroupImage: Bool = false
 
     let createBroadcast: Bool
+    let createEmail: Bool
```

Comment:
This createEmail and createBroadcast could probably be consolidated into an enum? Or am I misunderstanding and is it possible to have both creatBroadcast: true and createEmail: true?